libs/dfi/src/dyn_type.c

```diff
@@ -294,6 +296,14 @@ static int dynType_parseComplex(FILE *stream, dyn_type *type) {
     if (status == OK) {
         TAILQ_FOREACH(entry, &type->complex.entriesHead, entries) {
             count +=1;
+            struct complex_type_entry *entry2 = NULL;
```

Comment:
This is far from enough for runtime verification. If descriptors are generated from service header, then we need not to worry about this, since the compiler will check it for us.